packages/shared/src/components/modals/FeedFiltersModal.tsx

```diff
@@ -8,12 +8,20 @@ import useFeedSettings from '../../hooks/useFeedSettings';
 import { CustomSwitch } from '../fields/CustomSwitch';
 import { getFilterCardPreviews } from '../filters/FilterCardPreview';
 import { cloudinary } from '../../lib/image';
+import ThemeWidget from '../widgets/ThemeWidget';
 
 interface FeedFitlersModalProps extends ModalProps {
   trigger?: string;
 }
 
+const themes = [
+  { label: 'Dark', value: 'dark' },
+  { label: 'Light', value: 'light' },
+  { label: 'Auto', value: 'auto' },
+];
+
 function FeedFitlersModal(props: FeedFitlersModalProps): ReactElement {
+  const [selectedTheme, setSelectedTheme] = useState('');
```

Comment:
This in return could also cast as a theme type then.

Reply:
I think we should have "auto" as the default value here.

Reply:
Good point. Though the "dark" mode is at the top-most. Let me know whether we use that or push with `auto`.

Reply:
I think product hard decided on dark because we target devs and 90% has currently set to dark, feel free to raise it in product chatter though.